internal/pkg/cli/flag.go

```diff
@@ -110,8 +115,11 @@ Defaults to all logs. Only one of end-time / follow may be used.`
 	deleteSecretFlagDescription      = "Deletes AWS Secrets Manager secret associated with a pipeline source repository."
 	svcPortFlagDescription           = "Optional. The port on which your service listens."
 
-	storageFlagDescription        = "Name of the storage resource to create."
-	storageServiceFlagDescription = "Name of the service to associate with storage."
+	storageFlagDescription             = "Name of the storage resource to create."
+	storageServiceFlagDescription      = "Name of the service to associate with storage."
+	storagePartitionKeyFlagDescription = "Partition key for the DDB table. Must be of the format '<keyName>:<dataType>'."
+	storageSortKeyFlagDescription      = "Sort key for the DDB table. Must be of the format '<keyName>:<dataType>'."
+	storageAttributeFlagDescription    = "Attributes for a DDB table. Must be of the format '<name>:<dataType>'. Can be specified multiple times."
```

Comment:
Can we split these descriptions to multiple lines trying to stay <80 chars per line:
```suggestion
	storagePartitionKeyFlagDescription = `Partition key for the table.
Must be of the format '<keyName>:<dataType>'.`
	storageSortKeyFlagDescription      = `Sort key for the table. 
Must be of the format '<keyName>:<dataType>'.`
	storageAttributeFlagDescription    = `Attributes for the table. 
Must be of the format '<name>:<dataType>'. Can be specified multiple times.`
```